package.json

```diff
@@ -132,6 +131,10 @@
     "npm-user-validate": "1.0.1",
     "serialize-javascript": "6.0.0",
     "diff": "5.0.0",
-    "object-path": "0.11.8"
+    "object-path": "0.11.8",
+    "npm": "6.14.17"
+  },
+  "dependencies": {
+    "exceljs": "^4.3.0"
```

Comment:
Nit: move it to the dev dependencies.

PS: actually for the root `package.json`, we can simply have `dependencies`, no `devDependencies` needed.

Reply:
Okay, so are you suggesting I move all the packages within `devDependencies` to `dependencies`?

Reply:
Yes for monorepos that's usually the case for the root `package.json`. See https://github.com/commercetools/merchant-center-application-kit/blob/e40fcd6cda229fd8f7af8b33f6a5c3541f361aa0/package.json#L86

Reply:
Okay, cool thanks, will make the changes ASAP🙏🏽